web_src/js/index.js

```diff
@@ -179,4 +180,5 @@ onDomReady(() => {
   initRepoDiffView();
   initPdfViewer();
   initScopedAccessTokenCategories();
+  initActionsEditBtns();
```

Comment:
It could only be called on some related pages, eg: in some `initXxxx`

Reply:
2506e95 and 791983f?